webrender/src/resource_cache.rs

```diff
@@ -764,6 +764,24 @@ impl ResourceCache {
         debug_assert_eq!(self.state, State::QueryResources);
         self.state = State::Idle;
     }
+
+    pub fn clear_namespace(&mut self, namespace: IdNamespace) {
+        //TODO: use `retain` when we are on Rust-1.18
```

Comment:
We'll want to also remove the items from the texture cache, if they exist in there, I guess?